addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -3344,6 +3344,9 @@ options.registry.WebsiteAnimate = options.Class.extend({
         await this._super(...arguments);
```

Comment:
> I think task-3285817 is enough (as it means "related to task-XXX") or else create a new task and use its id 🙂

@sobo-odoo Thank's for the check, I think we can use "related to ..." (following: https://github.com/odoo/odoo/pull/134185#pullrequestreview-1630584746) But I just kept the `task-...` since the fix on `17.0` was a part of the task's diff :+1:

> Maybe add a ref to this commit 🙂    :ballot_box_with_check:  

> Also, just to be sure, the forward-port should stop before 17.0 then, right (since it is a backport of 17.0) ?

Actually, I will need to tweak the code anyway on 17.0+ (to set the class on the `.o_handles` element...) let's keep it "to master" and I will make necessary adjustments on fw-port :+1:

Reply:
Related to is for when the PR/commit is not directly done for the mentioned task/opw: it originated from there but it's not fixing the opw or doing the task spec.

`task-123` = code in charge of implementing spec of task 123
`opw-123` = code in charge of fixing the client ticket 123
`related to opw-123` or `linked to opw-123` or any wording you prefer = it's somehow involved in this opw but not fixing it directly, or it originates from this opw but not fixing it either. Same for tasks.

At the end, we just want things to be referenced for easy navigation and finding things back, so if you don't exactly follow this one time or the other there is no harm, but ideally it's the way to go.

Reply:
Okay I was not aware of that (I always create tasks for my PRs so I never encountered the situation) :open_mouth: 
Thanks for telling me :+1: